compiler/next/include/chpl/uast/Conditional.h

```diff
@@ -135,7 +135,7 @@ class Conditional final : public Expression {
   */
   ASTListIteratorPair<Expression> thenStmts() const {
     auto begin = children_.begin() + thenBodyChildNum_;
-    auto end = begin + numThenBodyStmts_ - 1;
```

Comment:
Looks like there are a few bug fixes here and in Loop - it'd be good to mention that the PR includes some bug fixes in the PR message.

Reply:
I think they already are (at least, that's why I remember not calling it out myself)

Reply:
Indeed they are! Thanks for pointing out that I had missed them.

> Also adjust the end() bounds of several statement body iterators by
removing the -1 at the end.